public/scripts/scripts.js

```diff
@@ -21,9 +21,7 @@ function generatePlaylistDropdown(playlists, allCallSongs){
   $('#playlistList').on('change', function() {
     $("#trackList").children().remove();
     var access_token = localStorage.getItem("access_token");
-    var playListTracks = getPlaylistTracks(access_token, allCallSongs);
```

Comment:
getPlaylistTracks doesn't have a useful return value, so this code is kind of misleading - what getPlaylistTracks does is kick off an ajax request and that's it, it doesn't hang around to collect data. so all it did was return `[]`, and that's not useful, so i deleted this playListTracks variable and console.log call. calling initializePlayListControl with the empty list probably did spooky stuff too, so i deleted that line.

since getPlaylistTracks is mainly called for its side effects, maybe a name like makePlaylistTracksRequest would be more clear, in order to prevent confusion by communicating that it doesn't have a useful return value (which is totally fine here, just something that's good to be clear about). up to you!